stylize/__main__.py

```diff
@@ -57,6 +57,9 @@ def main():
 
     # register any formatter-specific arguments
     for formatter in formatters:
+        if formatter.get_command() == "false":
+            print("[ERR] A required dependency was not found. Check to see if clang-format is available on your path.")
```

Comment:
It may be a little cleaner if we change get_command() to return None on failure.  That way we can avoid the usage of a false string here.